pyscriptjs/src/python/pyscript.py

```diff
@@ -136,6 +137,39 @@ def write(element_id, value, append=False, exec_id=0):
             )
         )
 
+    @classmethod
+    def set_version_info(cls, version_from_runtime: str):
+        """Sets the __version__ and version_info properties from provided JSON data
+        Args:
+            version_from_runtime (str): A "dotted" representation of the version:
+                YYYY.MM.m(m).releaselevel
+                Year, Month, and Minor should be integers; releaselevel can be any string
+        """
+
+        # __version__ is the same string from runtime.ts
+        cls.__version__ = version_from_runtime
+
+        # version_info is namedtuple: (year, month, minor, releaselevel)
+        version_parts = version_from_runtime.split(".")
+        version_dict = {
+            "year": int(version_parts[0]),
+            "month": int(version_parts[1]),
+            "minor": int(version_parts[2]),
+        }
+
+        # If the version only has three parts (e.g. 2022.09.1), let the releaselevel be ""
+        try:
+            version_dict["releaselevel"] = version_parts[3]
+        except IndexError:
+            version_dict["releaselevel"] = ""
+
```

Comment:
ah, this seems to contradict my previous comment. But then I don't understand how "normal" versions are supposed to like, with or without the `.final`?

Reply:
I am inclined towards `final` or `release` instead of `""` but have no strong opinions to be honest.